docs/source/overview/whatisit.rst

```diff
@@ -40,7 +40,7 @@ While Flight Solo provides single-user clusters, the Enterprise version of Fligh
 How much does it cost?
 ----------------------
 
-Alces Flight Compute is an open-source software appliance and is freely available to users, released under the General Public License (GPL) - please see the `Alces Flight EULA <https://s3-eu-west-1.amazonaws.com/flight-aws-marketplace/2017.1/AlcesFlight_2017.1_EULA.txt>`_ for details. The team at Alces package both a **Solo Community** version which is free to use, and a commercially supported **Solo Professional** version which customers can buy to get access to more features and professional assistance. Alces also provide multi-user clusters based on the **Flight Enterprise** version for both on-premise and public-cloud platforms - if you enjoy using Flight Compute Solo, contact Alces for more information on how you can buy your own multi-user environment.
+Alces Flight Compute is an open-source software appliance and is freely available to users, released under the General Public License (GPL) - please see the `Alces Flight EULA <https://s3-eu-west-1.amazonaws.com/flight-aws-marketplace/2017.1/AlcesFlight_2017.1_EULA.txt>`_ for details. The team at Alces package both a **Solo Community** version which is free to use, and a commercially supported **Solo Professional** version which customers can buy to get access to professional assistance. Alces also provide multi-user clusters based on the **Flight Enterprise** version for both on-premise and public-cloud platforms - if you enjoy using Flight Compute Solo, contact Alces for more information on how you can buy your own multi-user environment.
```

Comment:
This should probably point to a 2017.2 EULA (though I don't think they'd be any different).